developer/deploying/platform.md

```diff
@@ -0,0 +1,179 @@
+# Reaction Platform
+
+Reaction Commerce offers a managed deployment option, the [Reaction Platform](http://getrxn.io/reaction-platform).
+
+The `reaction-cli` incorporates functionality for any team to deploy Reaction to multiple environments. [Visit reactioncommerce.com](http://getrxn.io/reaction-platform) to get a demo or request an **invite token**.
+
```

Comment:
Add one summary sentence like, `In this guide, you'll learn how to use Launchdock to configure, deploy and set up continuous integration and testing from the command line.`

Reply:
👍 that's a good idea.  This section was mostly about pointing to the landing pages for signup to use the tool., so maybe also "before beginning this guide, you'll need to..."